docs/reference/cos.md

```diff
@@ -5,16 +5,22 @@
 ### Runner and Charm Insights
 Upon [COS](https://charmhub.io/topics/canonical-observability-stack) integration, this charm initiates the transmission of various metrics—refer to the relevant [specification](https://discourse.charmhub.io/t/specification-isd075-github-runner-cos-integration/12084) for comprehensive details—regarding the runner instances and the charm itself.
 
-The dashboard presents the following panels:
-
-- Lifecycle Counters: Tracks the frequency of Runner initialisation, start, stop, and crash events.
-- Idle Runners after Reconciliation: Reflects the count of Runners marked as idle during the last reconciliation event. Note: This data updates post-reconciliation events and isn't real-time.
-- Duration Observations: Each data point aggregates the last hour, showcasing minimum, maximum, and average durations for:
-    - Runner installation
-    - Runner idle duration
-    - Charm reconciliation duration
-    - Job duration
-    - Job queue duration - how long a job waits in the queue before a runner picks it up
+The dashboard presents the following rows:
+
+- General: Displays general metrics about the charm and runners, such as:
+  - Lifecycle Counters: Tracks the frequency of Runner initialisation, start, stop, and crash events.
```

Comment:
Picky but, why the capital leter on "Counters"?

Reply:
Its [title case style](https://en.wikipedia.org/wiki/Title_case). But can change that.

Reply:
If we follow the Ubuntu style guide, sentence case should be used for all headings: https://docs.ubuntu.com/styleguide/en/#headings , so removing title case here may make sense.